### ISSUE TITLE
Allow instance admin to set autoplay off by default

### ISSUE THREAD

**Issue by FediVideos** [Chocobozzz/PeerTube#6167]
### Describe the problem to be solved

I would rather have autoplay off on my instance, I find it annoying on other sites when videos play automatically.

At the moment if someone isn't logged in and they visit my instance on a laptop, videos play automatically. As far as I can tell there is no setting that allows the instance admin to switch off autoplay for people who aren't logged in?

### Describe the solution you would like

There should be a setting on the admin's GUI controls which allows admins to optionally set autoplay off by default.

**Comment by EricGuic** [comment]
Good idea, and if I'm not wrong, this settings is also required for complying with good practices for accessibility for people with disabilities.

**Comment by PARVNEMA** [comment]
i think can switch the autoplay  through javascript and videotag (if used)

**Comment by Khyvodul** [comment]
Hi,

I'm interested in working on this new feature request.

I'm wondering where is the best place to put this new settings in the admin's GUI:
Should it be under _Configuration > Basic > Videos_ or would you prefer it somewhere else ?

Also, should it be a
- positive setting: "Enable automatic video playback start by default", which would be set to true by default (when no user preference) to keep the behavior consistent with the current situation without this new setting
- negative setting: "Disable automatic video playback start by default", which would then be set to false by default to keep the behavior consistent with the current situation without this new setting ?

Thanks

**Comment by Chocobozzz** [comment]
Prefer to put the setting in "New user" section, because this is just the "default" setting when the user has not changed it.

Also prefer the "Enable..." form when using checkboxes

**Comment by Khyvodul** [comment]
Ok noted. I'll make the changes accordingly.

Thanks for the feedback

**Comment by Chocobozzz** [comment]
Implemented in https://github.com/Chocobozzz/PeerTube/pull/6788
